examples/semantic_segmentation/README.md

```diff
@@ -9,6 +9,14 @@ This sample demonstrates DL model compression capabilities for semantic segmenta
 - DataParallel and DistributedDataParallel modes
 - Tensorboard output
 
+## Installation
+
+To work with the sample you should install the corresponding Python package dependencies
+
+```
+pip install examples/requirements.txt
+```
+
```

Comment:
We already have this in root README.md. It doesn't hurt to give another hint at sample README level, though, but you should do this for each sample then.

Reply:
mb `pip  install **-r**  examples/requirement.txt` ?

Reply:
@vshampor It seems to be more convenient for users.
 I did for classification, detection and segmentation. Do we have more?

Reply:
> mb `pip install **-r** examples/requirement.txt` ?

Should be. Need to change main Readme

Reply:
@aleksei-kashapov please adjust the root README as well within the scope of this PR.

Reply:
Done